frontend/ui/elements/screen_notification_menu_table.lua

```diff
@@ -0,0 +1,175 @@
+                },
+                {
+                text = _("Bottom menu fine tuning"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_FINE)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_FINE)
+                end,
+                },
+                {
+                text = _("Bottom menu three dots"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_BOTTOM_MENU_MORE)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_BOTTOM_MENU_MORE)
+                end,
+                },
+                {
+                text = _("Gestures and profiles"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_DISPATCHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_DISPATCHER)
+                end,
+                },
+                {
+                text = _("Other sources"),
+                checked_func = function()
+                    return isMaskEnabled(Notification.SOURCE_OTHER)
+                end,
+                callback = function()
+                    toggleMask(Notification.SOURCE_OTHER)
+                end,
+                },
+            },
+        }
+    end
+    return ""
+end
+
+return {
+    text = _("Notification level"),
+    help_text = _([[KOReader may show notification popups at top of screen on various occasions.
+You can decide here what kind of notifications to show or hide.]]),
+    checked_func = function()
+        local value = G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+        return  value ~= 0
+    end,
+    sub_item_table = {
+        {
+        text = _("No popups"),
+        help_text = _("No popups are shown at all."),
+        checked_func = function()
+            return getMask() == Notification.SOURCE_NONE
+        end,
+        callback = function()
+            setMask(Notification.SOURCE_NONE)
+        end,
+        separator = true,
+        },
+        {
+        text = _("Only some bottom menu popus"),
+        help_text = _("Just show popups from the bottom menu, which don't have visual feedback."),
+        checked_func = function()
+            return band(getMask(), Notification.SOURCE_BOTTOM_MENU) == band(Notification.SOURCE_SOME, Notification.SOURCE_BOTTOM_MENU)
+        end,
+        callback = function()
+            if getMask() == Notification.SOURCE_ALL then
+                setMask(Notification.SOURCE_NONE)
+            end
+            setMask(bor(
+                    band(Notification.SOURCE_SOME, Notification.SOURCE_BOTTOM_MENU),
+                    band(getMask(), Notification.SOURCE_DISPATCHER)))
+        end,
+        },
```

Comment:
I'd probably get rid of that one (it doesn't make much sense to me to show stuff from a ConfigDialog knob but only if it's not a fine-tune?).